src/main/java/org/humancellatlas/ingest/submission/SpreadsheetDownloadJob.java

```diff
@@ -0,0 +1,11 @@
+package org.humancellatlas.ingest.submission;
+
+import lombok.Data;
+
+import java.time.Instant;
+
+@Data
+public class SpreadsheetDownloadJob {
```

Comment:
It's generate rather than download, right?

Reply:
you are right! i'll rename it